internal/services/warehouse/resource_warehouse_test.go

```diff
@@ -78,6 +78,19 @@ func TestUnit_WarehouseResource_Attributes(t *testing.T) {
 			),
 			ExpectError: regexp.MustCompile(`The argument "display_name" is required, but no definition was found.`),
 		},
+		// error - no required attributes (configuration)
+		{
```

Comment:
test must contain not only error path but happy path (CRUD with config/creation payload) as well.